Simbody/include/simbody/internal/Force_Custom.h

```diff
@@ -247,6 +247,13 @@ class SimTK_SIMBODY_EXPORT Force::Custom::Implementation {
     virtual bool dependsOnlyOnPositions() const {
         return false;
     }
+    /**
+     * Called during the realizeSubsystemDynamicsImpl method in GenralForceSubsystemRep, used to determine whether a custom force
+     * should be parallelized. By default, this method returns false and can be overwritten by the force subclass.
+     */
+    virtual bool isParallelByDefault() const {
```

Comment:
> forces that are expensive but not threadsafe should also not be parallelized

OK, could be something like `shouldParallelizeIfPossible()`. Then the comments would explain that the user should only override that method if (a) the force is computationally expensive, and (b) it is safe to parallelize. The name should reflect what the author can know.